test/support/builders/models/audit/ViewCisPeriodAuditBuilder.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package support.builders.models.audit
+
+import audit.ViewCisPeriodAudit
+import support.TaxYearProvider
+import support.builders.models.UserBuilder.aUser
+import support.builders.models.audit.ContractorDetailsAndPeriodDataBuilder.{aContractorDetailsAndPeriodDataFromCisDeductions, aContractorDetailsAndPeriodDataFromCisUserData}
+
+object ViewCisPeriodAuditBuilder extends TaxYearProvider {
+
+  val anInYearViewCisPeriodAudit: ViewCisPeriodAudit =
```

Comment:
No need to have two instances. The difference between this one and the anEndOfYearViewCisPeriodAudit is one field. Please create one instance and override it where you need it to comply with the test builder design pattern. It will help removing duplication,

Reply:
updated